src/GraphControl/Control/Grapher.cpp

```diff
@@ -246,6 +255,26 @@ namespace GraphControl
                 co_await TryUpdateGraph(keepCurrentView);
             }
         }
+
+        int valid = 0;
+        int invalid = 0;
+        for (Equation ^ eq : Equations)
+        {
+            if (eq->HasGraphError)
+            {
+                invalid++;
+            }
+            if (eq->IsValidated)
+            {
+                valid++;
+            }
+        }
+        if (!m_trigUnitsChanged)
```

Comment:
Maybe it makes more sense to only run a render pass when the trig units change instead of re-plotting, would that work?

Reply:
I think we have to re-plot because the values of the points change when the trig units change